LICENSE

```diff
@@ -0,0 +1,11 @@
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Please copy the full content of the license to this file. That will be much more clear and self-contained.

Reply:
This is the suggested format, as per apache 2.0 instructions. I was on the fence about it, but decided following their mandate is the way to go. 

Please consider this. If you still disagree, I will change it.

Reply:
For your reference: https://www.apache.org/licenses/LICENSE-2.0